langchain/src/vectorstores/milvus.ts

```diff
@@ -405,6 +405,9 @@ export class Milvus extends VectorStore {
     dbConfig: {
```

Comment:
This should instead be `dbConfig: MilvusLibArgs`

Reply:
I've updated.